oidc-service-impl/src/main/java/io/mosip/esignet/services/ConsentHelperService.java

```diff
@@ -192,7 +181,7 @@ private String generateSignedObject(OIDCTransaction transaction, ConsentDetail c
             jwsObject = new JWSObject(jwsHeader.toBase64URL(), Base64URL.encode(claimsSet.toJSONObject().toJSONString())
                     ,Base64URL.encode(signature) );
         } catch (ParseException e) {
-
+            log.error(e.getLocalizedMessage());
         }
         return jwsObject == null ? "": jwsObject.serialize();
```

Comment:
Instead of returning empty string can we throw an exception and handle the same in the caller method?